src/frontend/src/layouts/PageBlock.tsx

```diff
@@ -46,19 +46,26 @@ const PageBlock: React.FC<PageBlockProps> = ({ title, headerRight, children, sty
         }
       >
         {title && (
-          <Box sx={{ display: 'flex', flexDirection: 'row', mb: 1 }}>
-            <Typography variant="h5" sx={{ flexGrow: 1 }}>
-              {title}
+          <Box sx={{ mb: 1 }}>
+            <Typography variant="h5" sx={{ display: 'flex', justifyContent: 'space-between' }}>
+              {collapsed ? (
+                <div>
```

Comment:
So A couple things here, we want to reduce repeated code, so really the only thing that is chaning is the expand more icon, so we only want to include that in our ternary statement, otherwise everything is the same. We also try to stay away from using raw html, so instead of ps and hs and divs, we like to use typographys, so we can make a typography of the same variant as the parent, and then only run the ternary operator for the expand more icon and then have the title come after the ternary operator and enclose that in the new typography you just made.